packages/nc-gui/components/smartsheet/header/CellIcon.vue

```diff
@@ -0,0 +1,92 @@
+<script setup lang="ts">
```

Comment:
I removed the .vue file since it was an unused duplicate. Looks like the ts extension was used because no template/stylesheet/script sections were there - it's a pure ts file.